build-tools/automation/yaml-templates/run-sliced-nunit-tests.yaml

```diff
@@ -28,7 +28,7 @@ steps:
 
 - ${{ if eq(parameters.retryFailedTests, 'false') }}:
   # If we aren't using auto-retry logic, then this is just a simple template call
-  - template: run-nunit-tests.yaml
+  - template: /build-tools/automation/yaml-templates/run-nunit-tests.yaml@self
```

Comment:
Do you know why this was required? I didn't have to do this in AndroidX to find templates.

Reply:
I'm not sure what caused it, but I hit a dozen or so trigger failures early on in the migration process due to incomplete template paths or that missing `self` declaration.